dotnet/samples/KernelSyntaxExamples/README.md

```diff
@@ -238,10 +239,10 @@ Once the token is acquired, it is added to the HTTP authentication header via th
 
 To construct this provider, the caller must specify:
 
-- _Client ID_ � identifier of the calling application. This is acquired by [registering your application with the Microsoft Identity platform](https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).
-- _Tenant ID_ � identifier of the target service tenant, or �common�
-- _Scopes_ � permissions being requested
-- _Redirect URI_ � for redirecting the user back to the application. (When running locally, this is typically http://localhost.)
+- _Client ID_ – identifier of the calling application. This is acquired by [registering your application with the Microsoft Identity platform](https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).
```

Comment:
What is this character?

Reply:
No idea, I saw this too and tried to revert it but it didn't work :/ Let me try to commit the suggestion from here

Reply:
I think it was the emdash but should be fixed now!

Reply:
Looks like it has something to do with the encoding? Half the time it renders in GH, the other half it doesn't
![image](https://github.com/microsoft/semantic-kernel/assets/125500434/2376017b-542f-4c9a-984c-7e1ab2d52207)
![image](https://github.com/microsoft/semantic-kernel/assets/125500434/d89f5230-f4be-40fd-bc16-44e0b4bb70da)